static/app/components/sentryAppComponentIcon.tsx

```diff
@@ -46,28 +46,27 @@ const getFallbackIcon = (slug: string) => {
   }
 };
 
-const SentryAppIcon = ({sentryAppComponent: {sentryApp}}: Props) => {
-  return (
-    <Feature features={['organizations:sentry-app-logo-upload']}>
-      {({hasFeature}) => {
-        const selectedAvatar = sentryApp?.avatars?.find(({color}) => color === false);
-        const isDefault = !(selectedAvatar && selectedAvatar?.avatarType === 'upload');
-        return hasFeature ? (
-          <AvatarWrapper
-            isDark={ConfigStore.get('theme') === 'dark'}
-            isDefault={isDefault}
-          >
-            <Avatar size={20} sentryApp={sentryApp} isColor={false} />
-          </AvatarWrapper>
-        ) : (
-          getFallbackIcon(sentryApp.slug)
-        );
-      }}
-    </Feature>
-  );
-};
+/**
+ * Icon Renderer for SentryAppComponents with UI
+ * (e.g. Issue Linking, Stacktrace Linking)
+ */
+const SentryAppComponentIcon = ({sentryAppComponent: {sentryApp}}: Props) => (
+  <Feature features={['organizations:sentry-app-logo-upload']}>
+    {({hasFeature}) => {
+      const selectedAvatar = sentryApp?.avatars?.find(({color}) => color === false);
+      const isDefault = !(selectedAvatar && selectedAvatar?.avatarType === 'upload');
```

Comment:
Do you need the `selectedAvatar && ` if it's already using `?.` ?

Reply:
you're totally right, my mistake the `?`s always break my brain.